src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -1,10 +1,9 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Truck
- * Do not remove no-field constructor
- */
 public class Truck extends Machine {
+    private String typeExcavator;
+    private Integer sizeExcavator;
```

Comment:
to use these fields properly, please, create getters, setters and constructor with parameters